airflow/contrib/hooks/gcs_hook.py

```diff
@@ -162,24 +162,20 @@ def download(self, bucket, object, filename=None):
         :param filename: If set, a local file path where the file should be written to.
         :type filename: str
         """
-        service = self.get_conn()
-        downloaded_file_bytes = service \
-            .objects() \
-            .get_media(bucket=bucket, object=object) \
-            .execute()
+        client = self.get_conn()
+        bucket = client.get_bucket(bucket)
+        blob = bucket.blob(blob_name=object)
 
-        # Write the file to local file path, if requested.
         if filename:
-            write_argument = 'wb' if isinstance(downloaded_file_bytes, bytes) else 'w'
-            with open(filename, write_argument) as file_fd:
-                file_fd.write(downloaded_file_bytes)
-
-        return downloaded_file_bytes
+            blob.download_to_filename(filename)
+            self.log.info('File downloaded to %s', filename)
+        # MAX XCOM Size is 48KB
+        # https://github.com/apache/airflow/pull/1618#discussion_r68249677
+        if blob.size < 49344:
```

Comment:
Can you create a constant with this number? 
This limit appears in several places in the application, but with different values.
https://github.com/apache/airflow/blob/master/airflow/contrib/operators/gcs_download_operator.py#L85

Reply:
Done